add-ons/grafana/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Add-on
+An addon that will deploy grafana with hawkular and prometheus datasources.
+It requires to connect to hawkular-metrics, provided by management-infra addon.
+
+## Deploy grafana
+To deploy grafana do:
+
+```
+minishift addon apply grafana --addon-env namespace=grafana
+```
+
+_NOTE_: You should provide the namespace where it will be installed with the addon-env namespace.
+
+## Use grafana
+grafana will be available at:
+
+```
+$ minishift openshift service hawkular-grafana -n <namespace>
```

Comment:
Isn't `<namespace>` here is `grafana` as you provided above in deploy step?

Reply:
This is just an example. The note in the readme clarifies it.